examples/echo_bot.py

```diff
@@ -10,7 +10,7 @@
 logger = logging.getLogger(__name__)
 
 
-@dp.message(commands={"start"})
+@dp.message(commands=["start"])
```

Comment:
{}

Reply:
It makes no sense because of the [different types](https://github.com/aiogram/aiogram/blob/410350a7eaefad7c84a7ba44b0f5f8410c40deb5/aiogram/dispatcher/filters/command.py#L29) and [bruteforcing ](https://github.com/aiogram/aiogram/blob/410350a7eaefad7c84a7ba44b0f5f8410c40deb5/aiogram/dispatcher/filters/command.py#L90)commands in the filter

Reply:
Such a bad news for bot performance :(